include/pdes/serial/poisson_nitsche_interface.h

```diff
@@ -0,0 +1,292 @@
+      std::unique_ptr<GridTools::Cache<dim, spacedim>> space_cache;
+      std::unique_ptr<GridTools::Cache<dim, spacedim>> embedded_cache;
+
+      /**
+       * The coupling between the two grids is ultimately encoded in this
+       * vector. Here the i-th entry stores a tuple for which the first two
+       * elements are iterators to two cells from the space and embedded grid,
+       * respectively, that intersect each other (up to a specified tolerance)
+       * and a Quadrature object to integrate over that region.
+       *
+       *
+       */
+      std::vector<
+        std::tuple<typename dealii::Triangulation<dim, spacedim>::cell_iterator,
+                   typename dealii::Triangulation<dim, spacedim>::cell_iterator,
+                   dealii::Quadrature<spacedim>>>
+        cells_and_quads;
+
+
+      ParsedTools::FiniteElement<dim, spacedim> space_fe;
+
+      /**
+       * The actual DoFHandler class.
+       */
+      DoFHandler<dim, spacedim> space_dh;
+
+      /**
+       * According to the Triangulation type, we use a MappingFE or a MappingQ,
+       * to make sure we can run the program both on a tria/tetra grid and on
+       * quad/hex grids.
+       */
+      std::unique_ptr<Mapping<dim, spacedim>> mapping;
+      /** @} */
+
+      /**
+       * \name Linear algebra classes
+       * @{
+       */
+      AffineConstraints<double>    space_constraints;
+      SparsityPattern              sparsity_pattern;
+      SparseMatrix<double>         system_matrix;
+      Vector<double>               solution;
+      Vector<double>               system_rhs;
+      ParsedLAC::InverseOperator   inverse_operator;
+      ParsedLAC::AMGPreconditioner preconditioner;
+      /** @} */
+
+      /**
+       * \name Forcing terms and boundary conditions
+       * @{
+       */
+
+
+      ParsedTools::Constants constants;
+
+      /**
+       * The actual function to use as a forcing term.
+       */
+      ParsedTools::Function<spacedim> forcing_term;
+
+
+      /**
+       * This is the value we want to impose on the embedded domain.
+       *
+       */
+      ParsedTools::Function<spacedim> embedded_value;
+
+
+      /**
+       * The coefficient in front of the Nitsche contribution to the stiffness
+       * matrix.
+       *
+       */
+      ParsedTools::Function<spacedim> nitsche_coefficient;
+
+      /**
+       * The actual function to use as a exact solution when computing the
+       * errors.
+       * */
+      ParsedTools::Function<spacedim> exact_solution;
+
+
+      ParsedTools::BoundaryConditions<spacedim> boundary_conditions;
+
+      /**
+       * \name Output and postprocessing
+       * @{
+       */
+
+
+      ParsedTools::ConvergenceTable error_table;
+
+
+
+      /**
+       * Choosing as embedded space the square $[-.0.45,0.45]^2$ and as
+       * embedding space the square $[-1,1]^2$, with embedded value the
+       * function $g(x,y)=1$, this is what we get
+       * @image html Poisson_1_interface.png
```

Comment:
I wrote here to add an image of the solution, but I don't see how this can be included without adding an "image" folder and giving the corresponding path.

Reply:
you can put the image in the folder `doc/image_files', and doxygen will find it. :)